src/widgets/select.js

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { classNames } from "../common";
+
+import styles from "./select.css";
+
+export default class Select extends React.Component {
+  static get propTypes() {
+    return {
+      className: PropTypes.string,
+      value: PropTypes.string,
+      onSubmit: PropTypes.func.isRequired,
+      disabled: PropTypes.bool,
+      options: PropTypes.arrayOf(PropTypes.shape({
+        l10nId: PropTypes.string,
+        value: PropTypes.string,
+      })).isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      className: "",
+      disabled: false,
+    };
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: props.value,
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  // Little bit confusing, but when the value changes, we call props.onSubmit from here.
+  handleChange(evt) {
+    const value = evt.target.value;
+    this.setState({ value: value });
+    this.props.onSubmit(value);
+  }
+
+  focus() {
+    this.selectElement.focus();
+  }
+
+  render() {
+    const { className, options, disabled, ...props } = this.props;
+    return (
+      <select value={this.state.value}
```

Comment:
From past experience, trying to style `<select/>` to the degree our designs call for has been very problematic (maybe impossible).  I think this `<Select/>` component is a good abstraction over what we might ultimately do, though it's worth taking a bit to see if such a component exists out in the larger ecosystem.